experimental/packages/otlp-exporter-base/test/node/configuration/shared-env-configuration.test.ts

```diff
@@ -78,11 +78,11 @@ export function testSharedConfigurationFromEnvironment(
       sinon.assert.calledTwice(spyLoggerWarn);
       sinon.assert.calledWithExactly(
         spyLoggerWarn,
-        'Configuration: OTEL_EXPORTER_OTLP_METRICS_TIMEOUT is invalid, expected number greater than 0 (actual: NaN)'
```

Comment:
I would appreciate some guidance for the maintainer how to go about this. The `getStringFromEnv `-function itself outputs this message and then `shared-env-configuration.ts` tries to log a warning in one situation too. For now I have updated the check to use the warning created by the `getStringFromEnv `-function.

I am not sure if that is what we want, though.

Reply:
I'm assuming that if you didn't make this update, the test was failing. Is that correct?

Reply:
@maryliag Yes, that's correct. My understanding is because of the warn call in `get*FromEnv()` function

Reply:
Yes, you are correct on what is happening. And it is fine, IMO.  The specific format of warning messages are not a promised interface.

Reply:
+1 to Trent message